mcc/checker/checkers.py

```diff
@@ -79,13 +86,27 @@ def commas_dominant(value):
 
         return is_majority_commas, number_commas, number_nonalphanumeric
 
+    @staticmethod
+    def is_comma_delimited(value):
+        """
+        Determine if the provided string matches a regex for detecting
+        comma-delimination.
+
+        @param value string to check
+        @return True if the provided string matches the regex for comma-delimination,
+                False otherwise.
+        """
+        return bool(CheckCommaSeparated.COMMA_SEPARATED.match(value))
+
     def run_global(self, blueprint, value):
         if value is None:
             return self.error('does not exist')
 
-        dominant, _, _ = CheckCommaSeparated.commas_dominant(value)
+        comma_delimited = CheckCommaSeparated.is_comma_delimited(value)
+
+        comma_dominant, number_commas, number_nonalphanumeric = CheckCommaSeparated.commas_dominant(value)
 
-        if dominant:
+        if comma_delimited or comma_dominant or (number_commas == number_nonalphanumeric == 0):
```

Comment:
Why are we considering strings with zero commas/zero nonalphanumerics to be csv's?

Doesn't seem to be the old behavior - https://github.com/podaac/mcc/blob/7aac198b90d67807ca6a2ad070222aa1ec8841f7/mcc/checker/checkers.py#L78C30-L78C68

Reply:
I was considering the case where a dataset defines a `keywords` field (which the checker examines for comma-separation) but the field consists of only a single keyword with no other special characters. I figured we would not want to flag that as failed since it is "technically" comma-separated.